test/unit/config/config-security.test.js

```diff
@@ -226,3 +226,13 @@ tap.test('#applyLasp', (t) => {
     t.end()
   })
 })
+
+tap.test('ai_monitoring should not be enabled in HSM', (t) => {
+  const config = Config.initialize({
+    high_security: 'true'
```

Comment:
this is hard to tell that it's working because currently ai_monitoring.enabled defaults to false. it'll eventually be true but can you update the test to set it to true to start?

Reply:
This is still incorrect because it's ai_monitoring.enabled

Reply:
Yeah, I realized that while out running my errands. It should be correct now.